appcenter-data/android/src/main/java/com/microsoft/appcenter/reactnative/data/AppCenterReactNativeDataPackage.java

```diff
@@ -0,0 +1,33 @@
+package com.microsoft.appcenter.reactnative.data;
+
+import android.app.Application;
+
+import com.facebook.react.ReactPackage;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.uimanager.ViewManager;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class AppCenterReactNativeDataPackage implements ReactPackage {
+
+    private AppCenterReactNativeDataModule mDataModule;
+
+    public AppCenterReactNativeDataPackage(Application application) {
+        this.mDataModule = new AppCenterReactNativeDataModule(application);
+    }
+
+    @Override
+    public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
+        return Collections.emptyList();
+    }
+
+    @Override
+    public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
+        List<NativeModule> modules = new ArrayList<>();
+        modules.add(mDataModule);
+        return modules;
+    }
```

Comment:
Missing this:
```
    // No @Override to support applications using React Native 0.47.0 or later
    public List<Class<? extends JavaScriptModule>> createJSModules() {
        return Collections.emptyList();
    }
```

Reply:
To clarify, `createJSModules()` method was deprecated in RN 0.47.0 but we keep it for backward compatibility with older RN versions.

So yes, please do as @guperrot suggested.

Reply:
Done